CONTRIBUTING.md

```diff
@@ -60,89 +60,74 @@ before submitting a PR.
 
 By contributing, you agree that your contributions will be licensed under its Apache License 2.0.
 
-# Building the solution
+## Preparing for a New Release
 
-The solution uses several open source software tools:
+The client uses generated stubs from upstream Qdrant proto definitions, which are downloaded from [qdrant/qdrant](https://github.com/qdrant/qdrant/tree/master/lib/api/src/grpc/proto).
 
-## Docker
+The generated files do not form part of the checked in source code. Instead, they are generated
+and emitted into the `src/Qdrant.Client/obj/Release`, and included in compilation.
 
-Qdrant docker image is used to run integration tests. Be sure to 
-[install docker](https://docs.docker.com/engine/install/) and have it running when running tests.
+### Pre-requisites
 
-## Bullseye
+Ensure the following are installed and available in the `PATH`.
 
-[Bullseye](https://github.com/adamralph/bullseye) is used as the build automation system for the solution.
-To get started after cloning the solution, it's best to run the build script in the root
+- [Dotnet 6.0.x](https://dotnet.microsoft.com/download/dotnet/6.0)
+- [Docker](https://docs.docker.com/engine/install/) for tests
 
-for Windows
+### Steps
 
-```
-.\build.bat
-```
+1. Set the `<QdrantVersion>` value in [Directory.Build.props](https://github.com/qdrant/qdrant-dotnet/blob/main/Directory.Build.props) to `dev`. In order to download the `dev` Docker image for testing and use the `dev` branch for fetching the proto files.
+
+2. Download and generate the latest client stubs by running the following command from the project root:
 
-for OSX/Linux
+For Windows
 
+```bash
+.\build.bat build
```

Comment:
The build exe that this invokes only downloads files from a given `<QdrantVersion>` once

https://github.com/qdrant/qdrant-dotnet/blob/e30739fd4ba928a2684170a6243a4e98b9123404/build/Main.cs#L56-L60
 

For a constantly changing upstream branch like `dev`, may want to either

- introduce a command line argument to delete the downloaded proto files

or

- introduce a convention to delete downloaded proto files for given `<QdrantVersion>`s

Reply:
Thanks.
Added a new flag.
[`bd1c1ae` (#69)](https://github.com/qdrant/qdrant-dotnet/pull/69/commits/bd1c1ae701d74c5e2e5b8993bb72a9703cac5444)